test/integration/crud/misc_cursors.test.js

```diff
@@ -1887,52 +1887,6 @@ describe('Cursor', function () {
     }
   );
 
-  it('shouldAwaitData', {
```

Comment:
This test started failing with my changes.  I started looking at it, and it doesn't even test what the test description seems to imply its testing.